### PR TITLE
feat(base): creates project base and server

### PR DESCRIPTION
**Pull request by robintsunny**
what: SFMovies set up
why: Building my own API

### CODE REVIEW

File: config/production.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = {
+  FOO: 'bar'
```
Comment (jesscxu) [inline config/production.js:4]: just an example, right? think you can remove it if it doesn't error for being empty!

**Review by jesscxu — APPROVED** [review]
LGTM, just add `.DS_Store` to `.gitignore` file

**Comment by jesscxu** [comment]
~also remember to make the PR commit messages follow the format listed on the guidelines! for ex the `feat(...)` format and~ 

EDIT: jk noticed you did do that, just make the PR title match that commit message (right now it says Base)

File: config/production.js (hunk 1 above)
Comment (ajpawlicki) [reply config/production.js:4]: Is this necessary?

File: lib/index.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const Util = require('util');
+
+const Config = require('../config');
+const Movies = require('./server');
```
Comment (ajpawlicki) [inline lib/index.js:6]: Did you decide to name this `Movies` or is that in the instructions?

**Review by ajpawlicki — APPROVED** [review]
lgtm

File: lib/index.js (hunk 2 above)
Comment (robintsunny) [reply lib/index.js:6]: it was in the instructions

Comment (robintsunny) [reply lib/index.js:6]: i think because it's the movie server, correct?

**Pull request merged**